docs/public/_redirects

```diff
@@ -511,6 +511,11 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /material-ui/experimental-api/css-theme-variables/usage/ /material-ui/customization/css-theme-variables/usage/ 301
 /material-ui/experimental-api/css-theme-variables/customization/ /material-ui/customization/css-theme-variables/configuration/ 301
 /base-ui/ https://base-ui.com/ 301
+#2025
+# TODO v7 change mui.com to v6.mui.com when domains are switched
```

Comment:
@DiegoAndai do we have some pattern for comments to search for before the stable release?

Reply:
I'll search for "todo v7" case insensitive. But honestly, at this point, we shouldn't do any further code changes so we can have a safe stable release